packages/create-app/templates/default-app/packages/backend/src/plugins/search.ts

```diff
@@ -20,7 +21,10 @@ export default async function createPlugin({
   // particular collator gathers entities from the software catalog.
   indexBuilder.addCollator({
     defaultRefreshIntervalSeconds: 600,
-    collator: DefaultCatalogCollator.fromConfig(config, { discovery }),
+    collator: DefaultCatalogCollator.fromConfig(config, {
```

Comment:
Thanks for covering this in `create-app`!  Could you also make the corresponding changes in the docs?  Believe it's just a couple of spots:

- [Search TechDocs How-To Guide](https://github.com/backstage/backstage/blob/master/docs/features/search/how-to-guides.md?plain=1#L74)
- [Search Getting Started Backend](https://github.com/backstage/backstage/blob/master/docs/features/search/getting-started.md?plain=1#L163)
- A few parts in the [Search Getting Started Backend Customization](https://github.com/backstage/backstage/blob/master/docs/features/search/getting-started.md?plain=1#L283)